### DISCUSSION TITLE
change of focus on windows

### DISCUSSION THREAD

**Discussion by fungilife** [labwc/labwc#2043] [Q&A]
I believed the answer would have been in a setting of rc.xml but all changes I made didn't alter the behavior (restarted or reconfigured tested) 

Say you work on an application which automatically parses its .conf file and passes the new settings, and you edit this conf file from a terminal.  As soon as you hit save the focus changes to that application and leaves terminal, and although I suspect most people want it that way, application starting shift to it automatically, I prefer to choose the focus manually than the wm choosing for me.

Is this a bug or a configuration I haven't located yet?

**Comment by johanmalm** [comment]
I'm not sure I understand this one. Application shouldn't be able to grab the focus by themselves.
There is no config option relating to this.
Could you give some steps to reproduce?

**Comment by Consolatis** [comment]
@fungilife 
I assume this is the issue you referred to in #2742? If yes, it wasn't closed but waiting for feedback.

**Comment by Narrat** [comment]
Maybe a reference on the focus topic.
I had a similar point when I asked about behaviour of my firefox and thunderbird window rules.
https://github.com/labwc/labwc/discussions/1987#discussioncomment-10065139
(And I vaguely remember it referencing in another issue...?)

And kinda related? At least similar but more extreme.
https://github.com/labwc/labwc/issues/1057
